examples/todo_app/lib/todo_list.dart

```diff
@@ -54,14 +55,17 @@ class TodoList extends StatelessWidget {
               icon: Icon(todo.done ? Icons.clear : Icons.check),
               onPressed: () {
                 var newTodo = todo.copyWith(done: !todo.done);
+                Hive.box('todos')
+                    .deleteAt(Hive.box('todos').values.toList().indexOf(todo));
```

Comment:
Why did you replace `delete(todo.id)` with `deleteAt(Hive.box('todos').values.toList().indexOf(todo))`?

Deleting by key is much faster than your solution. You have to copy all todo items to a list, find the one to be deleted. Hive will then internally walk through all the values again to find the key of the todo item.

Reply:
I tried it with the delete solution, it didn't work on my Pixel 3 XL emulator

Reply:
Strange. I'll test it on the emulator...

Reply:
Web version works for me, but when I restart the app on my emulator, it shows some weird behavior !
https://imgur.com/a/GPRXCQ2

Reply:
Thanks, I'll look into it.